src/react/connect.ts

```diff
@@ -45,12 +45,14 @@ export const useStream = <T>(
  * provided by the stream. In these cases the values from the stream will
  * override any values passed as props.
  *
- * @param WrappedComponent Component that will recieve props from the stream
+ * @param Component Component that will recieve props from the stream
  * @param stream$ Stream that will provide props to the component
- * @see useStream
+ * @returns A component that renders the original component with props retrieved
+ *  from the stream
+ * @see {@link useStream}
  */
 export const connect =
-  <Props extends {}, Observed extends Partial<Props>>( // eslint-disable-line @typescript-eslint/ban-types
+  <Props extends Record<string, unknown>, Observed extends Partial<Props>>(
```

Comment:
out of curiousity, why?

Reply:
This will restrict `Props` to be objects (so you can no longer pass primitives like a number or a boolean), and it means we don't need to have the lint ignore line there.